src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -9,8 +9,8 @@
       "requireClientSecret": true,
       "secret": "SampleClientSecret",
       "requireConsent": false,
-      "redirectUrls": [ "http://localhost:49176/signin-oidc" ],
-      "postLogoutRedirectUrls": [],
+      "redirectUrls": [ "http://localhost:8072/signin-oidc" ],
```

Comment:
Why 8072? It looks like `launchSettings.json` still defaults to 49176 for the sample MVC client.

It might be worth looking at the default `launchSettings.json` port settings across the sample client, resource, and IDS projects as I have to change them every time I want to debug locally – unless Stuart is planning to do so as part of his dev environment changes...

Reply:
Sorry, just spotted that it's the port configured in `docker-compose.yml`.

(Although that doesn't resolve the port issue when not using the dockerized version.)

Reply:
This was used during manual testing. Stuart said he would be changing these shortly. So I left it. I'll change them back to match the launch settings of the MVC client e.g. 49176